action.yml

```diff
@@ -20,10 +20,23 @@ inputs:
   private_api_key:
     description: ''
     required: true
+  working_dir: # used for s2i-core and s2i-base images only
+    description: ''
+    required: false
 
 runs:
   using: 'composite'
   steps:
+    - name: Set working dir
+      id: working_dir
+      shell: bash
+      run: |
+        if ${{ inputs.working_dir  == "" }}; then
+          echo "path=${{ inputs.version }}" >> "$GITHUB_OUTPUT"
```

Comment:
This could be by default like:
```
echo "path=${{ inputs.version }}" >> "$GITHUB_OUTPUT
if <condition; then
     echo "path=${{ inputs.working_dir }}" >> "$GITHUB_OUTPUT"
fi
```